app/views/project_approvals/new.html.erb

```diff
@@ -0,0 +1,9 @@
+<h3><%= pending_project.title %></h3>
```

Comment:
Let's not be afraid of using H1s! This is an accessibility note. (Nonblocking)

Reply:
Good point! Fixed